src/DependencyInjection/EventBusExtension.php

```diff
@@ -37,6 +38,10 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
 
         if ($mergedConfig['logging']['enabled']) {
             $loader->load('event_bus_logging.yml');
+
+            $container->getDefinition('simple_bus.event_bus.notifies_message_subscribers_middleware')
+                ->replaceArgument(1, new Reference('logger'))
+                ->replaceArgument(2, '%simple_bus.event_bus.logging.level%');
```

Comment:
@matthiasnoback Is there actually a reason to use a container parameter instead of a configuration value?